components/GoalDetailCard/index.js

```diff
@@ -14,10 +14,79 @@ const StyledForm = styled.form`
   align-items: center;
   justify-content: center;
   flex-direction: column;
-  margin-bottom: 75px;
+  margin-bottom: 60px;
   min-height: 100vh;
 `;
 
+const StyledGoalDetailWrapper = styled.main`
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  flex-direction: column;
+  margin-bottom: 40px;
+`;
+
+const StyledHeadline = styled.h2`
+  font-size: 24px;
+  color: #272727;
+  background-color: #eee9da;
+`;
+
+const StyledDescriptionList = styled.dl`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+  margin: 0;
+`;
+
+const StyledDescriptionContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  min-width: 700px;
+  max-width: 800px;
+  height: 40px;
+  background-color: #6096b4;
+
+  @media (min-width: 375px) and (max-width: 768px) {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    min-width: 280px;
+    max-width: 800px;
+    height: 40px;
+    background-color: #6096b4;
+  }
+`;
+
+const StyledDescription = styled.dt`
+  background-color: #6096b4;
+  font-weight: bold;
+  letter-spacing: 1px;
+  color: #fcfbf4;
+  align-items: center;
+`;
+
+const StyledDescribedItemContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 55px;
+  background-color: #eee9da;
+`;
+
+const StyledDescribedItem = styled.dd`
+  background-color: #eee9da;
+  margin: 10px 0 10px 0;
```

Comment:
You could also write it this way:
```suggestion
  margin: 10px 0;
```